.github/workflows/stale.yml

```diff
@@ -21,21 +21,20 @@ jobs:
     steps:
       - uses: actions/stale@v6
         with:
-          debug-only: false
+          debug-only: true
```

Comment:
Did we decide to go with `debug` after all? I thought everyone was against it.

Not that I mind, but I don't want to sneak it in behind anyone's back.

Reply:
We decided to **not use debug**.

Reply:
> Did we decide to go with `debug` after all?

I decided to enable it so it could run on debug during the weekend, but well... I will revert it then.